test/utils/bytecode.hpp

```diff
@@ -242,6 +244,25 @@ inline bytecode jumpi(bytecode target, bytecode condition)
     return condition + target + OP_JUMPI;
 }
 
+inline bytecode rjump(int16_t offset)
+{
+    return OP_RJUMP + big_endian(offset);
```

Comment:
This doesn't need the `bytecode{}` wrapping around `big_endian`, but the others below do?

Reply:
Unfortunately in this case + operator is not ambiguous  because we are adding `Opcode : uint8_t` but below in the second addition we are adding `bytecode` and `bytes` (assuming bytecode{} removal) and this operator is ambiguous. 

```
/Users/rodia/projects/evmone/test/utils/bytecode.hpp:252:34: error: use of overloaded operator '+' is ambiguous (with operand types 'bytecode' and 'typename std::enable_if_t<std::is_same_v<short, uint16_t> || std::is_same_v<short, int16_t>, bytes>' (aka 'std::basic_string<unsigned char>'))
    return condition + OP_RJUMPI + big_endian(offset);
           ~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/string:4348:1: note: candidate function [with _CharT = unsigned char, _Traits = std::char_traits<unsigned char>, _Allocator = std::allocator<unsigned char>]
operator+(basic_string<_CharT, _Traits, _Allocator>&& __lhs, basic_string<_CharT, _Traits, _Allocator>&& __rhs)
^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/string:4332:1: note: candidate function [with _CharT = unsigned char, _Traits = std::char_traits<unsigned char>, _Allocator = std::allocator<unsigned char>]
operator+(basic_string<_CharT, _Traits, _Allocator>&& __lhs, const basic_string<_CharT, _Traits, _Allocator>& __rhs)
^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/string:4340:1: note: candidate function [with _CharT = unsigned char, _Traits = std::char_traits<unsigned char>, _Allocator = std::allocator<unsigned char>]
operator+(const basic_string<_CharT, _Traits, _Allocator>& __lhs, basic_string<_CharT, _Traits, _Allocator>&& __rhs)
^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/string:1739:25: note: candidate function [with _CharT = unsigned char, _Traits = std::char_traits<unsigned char>, _Allocator = std::allocator<unsigned char>]
    friend basic_string operator+<>(const basic_string&, const basic_string&);
                        ^
/Users/rodia/projects/evmone/test/utils/bytecode.hpp:48:17: note: candidate function
inline bytecode operator+(bytecode a, bytecode b)
                ^

```

Reply:
Maybe this would also work in the case below
```
return condition + (OP_RJUMPI + big_endian(offset));
```

Reply:
Done.